src/ui/FileViewer/index.tsx

```diff
@@ -138,23 +108,50 @@ export const FileViewerComponent = ({
           )
         }
       </div>
+      <ImagesView props={props} stringSet={stringSet}/>
     </div>
   );
 };
 
 export interface FileViewerProps {
   message: FileMessage;
+  onClose: (e: MouseEvent) => void;
   isByMe?: boolean;
-  onClose: (e: MouseEvent<HTMLDivElement>) => void;
-  onDelete: (e: MouseEvent<HTMLDivElement>) => void;
+  onDelete?: (e: MouseEvent) => void;
+  currentIndex?: number;
+  onClickLeft?: () => void;
+  onClickRight?: () => void;
 }
 
 export default function FileViewer({
   message,
-  isByMe = false,
   onClose,
+  isByMe = false,
   onDelete,
+  currentIndex,
+  onClickLeft,
+  onClickRight,
 }: FileViewerProps): ReactElement {
+  if (message?.isMultipleFilesMessage()) {
+    return (
+      <FileViewerComponent
+        profileUrl={message?.sender?.profileUrl}
+        nickname={message?.sender?.nickname}
+        viewerType={ViewerTypes.MULTI}
+        fileInfoList={message?.fileInfoList.map((fileInfo: UploadedFileInfo): FileInfo => {
+          return {
+            name: fileInfo.fileName,
+            type: fileInfo.mimeType,
+            url: fileInfo.url,
+          };
+        })}
+        currentIndex={currentIndex}
+        onClickLeft={onClickLeft}
+        onClickRight={onClickRight}
```

Comment:
Should add default values to these props

Reply:
<img width="590" alt="Screen Shot 2023-08-18 at 2 35 35 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/11382805/c8bb9d95-1a7a-4072-8daf-6d1db8ed5012">
Same for `name` and `type` above

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/711/commits/5bc750b773df91faddbff1024879a475a6e08c08 applied here